tests/js/utils.js

```diff
@@ -190,6 +196,30 @@ export const provideUserInfo = ( registry, extraData = {} ) => {
 	} );
 };
 
+/**
+ * Registers the given module with settings to the given registry.
+ *
+ * @since n.e.x.t
+ * @private
+ *
+ * @param {Object}   registry    Registry object to dispatch to.
+ * @param {Object[]} [extraData] List of module objects to be merged with defaults. Default empty array.
+ */
+export const provideModuleRegistrations = ( registry, extraData = [] ) => {
+	const moduleIconMap = {
+		adsense: AdsenseIcon,
+		analytics: AnalyticsIcon,
+		optimize: OptimizeIcon,
+		'pagespeed-insights': PagespeedInsightsIcon,
+		'search-console': SearchConsoleIcon,
+		tagmanager: TagManagerIcon,
+	};
```

Comment:
Following-up on my previous comment: While this works for this particular issue, I meant that it would make sense to provide the complete **real** module registration args here for each of the six modules, which include not only the icon, but also a couple of JS components. I'm gonna add these changes here for now.